test/parallel/test-tls-parse-cert-string.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const tls = require('tls');
+
+const singles = 'C=US\nST=CA\nL=SF\nO=Joyent\nOU=Node.js\nCN=ca1\n'
```

Comment:
s/Joyent/Node.js Foundation/?  That also tests if blanks in the O field are parsed correctly.